blackbox/docs/admin/system-information.rst

```diff
@@ -961,6 +961,26 @@ CrateDB also logs both of these tables.
 By default, these tables are not populated, because tracking jobs and
 operations adds a slight performance overhead.
 
+The tracking jobs are stored in memory. The amount of memory that can be
+used to store the jobs is limited by the
+:ref:`Jobs Log Circuit Breaker <stats.breaker.log.jobs.limit>` and
+:ref:`Operations Log Circuit Breaker <stats.breaker.log.operations.limit>`.
+If the memory limit is reached, an error message will be logged and the log
+table will be cleared completely.
+
+You can avoid tripping these circuit breakers by configuring a limit,
```

Comment:
this seems like a repetition of the previous sentence

Reply:
hmm, I disagree. We previously talk about the circuit breakers tripping and clearing the tables and this paragraph talks about setting size/eviction limits in order to truncate the tables so we avoid tripping the breakers.

do you think we should rephrase it somehow?

Reply:
well it seems to me (as an uneducated user) that the first says "if you hit a limit the tables are truncated" and the second one says "you can avoid this by configuring a limit at which the tables are truncated"

it's not obvious that we're talking about two different limits and what the benefit of one being responsible for table truncation vs the other

does that make sense?

Reply:
The first one does not say "if you hit a limit the tables are truncated" but rather -> If the memory limit is reached, an error message will be logged and the log table will be cleared completely.

I'm pasting the entire paragraph here for full reference 

```
 The tracking jobs are stored in memory. The amount of memory that can be used to store the jobs is limited by the :ref:`Jobs Log Circuit Breaker <stats.breaker.log.jobs.limit>` and :ref:`Operations Log Circuit Breaker <stats.breaker.log.operations.limit>`. If the memory limit is reached, an error message will be logged and the log table will be cleared completely.

You can avoid tripping these circuit breakers by configuring a limit, after which the logs tables will be truncated.

One option is to configure how many records are to be stored in the logs tables. This is configurable using :ref:`Jobs Log Size <stats.jobs_log_size>` and :ref:`Operations Log Size <stats.operations_log_size>`.
```

The first paragraph discusses the _memory_ limit. Generally, the circuit breakers are tripped when certain memory limits are hit. When this memory limit is hit " the log table will be cleared completely". 

Ok, the second paragraph offers options to avoid tripping the circuit breakers (ie. avoiding "the log table will be cleared completely") using 2 options ... limiting the number of items in the table or using an eviction time. 

I'm not really sure if you think this is a wording issue or a structural issue?

Reply:
I think it's a wording issue

I would suggest that in the second paragraph you mention that you can avoid the out-of-memory situation by setting a number-of-items limit or a record eviction time

I would also maybe include a brief explanation why, as someone operating cratedb, you care, one-way-or-another, what causes record eviction

am I right in thinking that:

- if a max records limit is configured, then the table will prune the oldest records to make way for the newest, and so will never be truncated

- if an eviction time is specified, similarly, the tables will usually be full of records. and you don't risk them being truncated

if I have understood this correctly, I would make this more clear in the text

Reply:
@nslater pushed fixup